src/main/kotlin/com/sourcegraph/cody/chat/ExportChatsBackgroundable.kt

```diff
@@ -40,6 +41,7 @@ class ExportChatsBackgroundable(
             .filter { it.internalId != null }
             .filter { chat -> if (internalId != null) chat.internalId == internalId else true }
 
+    var dummyInternalId = UUID.randomUUID().toString()
```

Comment:
Does it also work for restore all?
Or, in other words, we just need one `dummyInternalId` or one per restored chat?

Reply:
we need one per chat and that's what I do here (see line 58)